challenge-02/README.md

```diff
@@ -58,8 +71,22 @@ Crie uma função com as seguintes características:
 5. Se nenhum argumento for passado, retorne o valor booleano `false`.
 6. E ainda, se nenhuma das condições acima forem atendidas, retorne `null`.
 */
-?
+function calculo2(x, y, z){
+	if(y === undefined && z === undefined){
```

Comment:
-1
Aqui você verifica se o segundo ou o terceiro parâmetros são passados, mas e se o primeiro parâmetro também não for passado, a função vai tentar retornar o valor desse primeiro parâmetro, e a resposta vai ser `undefined`.

Você precisa verificar pelos 3 parâmetros para ter certeza de que realmente só o primeiro está sendo passado.